common/persistence/client/targeted_fault_injection.go

```diff
@@ -100,18 +100,19 @@ func (d *dataStoreErrorGenerator) Generate() error {
 
 // newError returns an error based on the provided name. If the name is not recognized, then this method will
 // panic.
-func newError(errName string, errRate float64) error {
+func newError(errName string, errRate float64, methodName string) error {
+	header := fmt.Sprintf("fault injection error at %s with %.2f rate", methodName, errRate)
```

Comment:
Do we really care about including the rate here? We can look at the config to find it

Reply:
yeah, may be not. I just found it useful while reading logs.